Core/Services/Auth/AuthServicesProvider.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Core.Interfaces.Auth;
+using Core.Interfaces.Email;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.Extensions.Configuration;
+using Storage.Identity;
+
+namespace Core.Services.Auth
+{
+    class AuthServicesProvider
```

Comment:
Is the default internal access modifier correct for this class?

Reply:
Yes, because I use this class only in this folder